examples/error-binding-pattern/error_binding_pattern.md

```diff
@@ -0,0 +1,12 @@
+# Error Binding Pattern
+
+An error binding pattern matches the different elements of an error value such as the errors message, error cause and error detail mapping. It can also be used to destructure the reason, cause, and the detail mapping values from an error value, and assign them to existing variable references.
```

Comment:
```suggestion
An error binding pattern matches the different elements of an error value such as the error message, error cause, and error detail mapping. It can also be used to destructure the reason, cause, and detail mapping values from an error value, and assign them to the existing variable references.
```